.circleci/config.yml

```
@@ -0,0 +1,45 @@
+version: 2.1
+
+jobs:
+  tests-py36:
+    docker:
+      - image: 'cimg/python:3.6'
+    steps:
+      - checkout
+      - run: pip install --user tox
+        # I didn't find a way using TOXENV with regular expression, similar to TOX_SKIP_ENV
+        # https://tox.readthedocs.io/en/latest/config.html#conf-envlist
```

```suggestion
        # Select only the environments that match the Python version we have installed 
        # in this Docker image (e.g. `py36-*`). I didn't find a way using TOXENV with
        # regular expression for this. Read more at
        # https://github.com/tox-dev/tox/issues/238 and
        # https://tox.readthedocs.io/en/latest/config.html#conf-envlist
```

@ericholscher looks better?